activesupport/lib/active_support/test_case.rb

```diff
@@ -149,5 +149,9 @@ def parallelize_teardown(&block)
     alias :assert_not_same :refute_same
 
     ActiveSupport.run_load_hooks(:active_support_test_case, self)
+
+    def inspect # :nodoc:
+      "#<#{self.class.name}:#{'%#016x' % object_id}>"
```

Comment:
No strong opinion either way (non blocking). How do you feel about this vs.

``` suggestion
      Object.instance_method(:to_s).bind_call(self)
```

Reply:
Isn't that just `alias_method :inspect, :to_s`?

Not sure if I should be nitpicking in a quick fix PR 🤣